.pre-commit-hooks.yaml

```
@@ -4,5 +4,6 @@
   entry: black
   language: python
   language_version: python3
+  minimum_pre_commit_version: '2.9.0'
```

```suggestion
  minimum_pre_commit_version: "2.9.0"
```
my bad, sorry, `prettier` wants double quotes